charts/metrics-server/templates/apiservice.yaml

```diff
@@ -5,10 +5,18 @@ metadata:
   name: v1beta1.metrics.k8s.io
   labels:
     {{- include "metrics-server.labels" . | nindent 4 }}
+{{- if .Values.certManager.enabled }}
+  annotations:
+    certmanager.k8s.io/inject-ca-from: {{ printf "%s/%s-root-cert" .Release.Namespace (include "metrics-server.fullname" .) | quote }}
```

Comment:
Based on the [cert-manager documentation](https://cert-manager.io/docs/concepts/ca-injector/) I don't think this annotation is needed. I can't see when the legacy annotations were removed but it appears to be pre v1.